client/webserver/site/src/html/orders.tmpl

```diff
@@ -35,6 +35,9 @@
         {{end}}
         <div class="apply-bttn d-hide"><button class="bg2 demi">apply</button></div>
       </div>
+      <div id="exportOrders" class="filter-display bg1">
+        <a class="ico-open fs14 pointer plainlink"></a> Export
+      </div>
```

Comment:
![image](https://user-images.githubusercontent.com/6109680/103639269-baaffa00-4f13-11eb-9115-06a32e049887.png)

You've styled it as if it's a header. Please style it as if it's a button similar to other buttons in the client.